dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntitiesExportControllerTest.java

```diff
@@ -142,12 +142,15 @@ void setUp() throws IOException {
 
     importUser = userService.getUser("tTgjgobT1oS");
     injectSecurityContextUser(importUser);
+    setUpToBeMigrated();
 
-    trackerObjects = testSetup.importTrackerData();
+    manager.flush();
+    manager.clear();
     testSetup.importTrackerData("tracker/tracker_multi_text_attribute_data.json");
 
     manager.flush();
     manager.clear();
+    trackerObjects = testSetup.importTrackerData();
```

Comment:
All of the above code movement is caused by a mix between Hibernate queries and JDBC queries, which prevents data from being flushed to the database, so the delete execution cannot be performed.